ballet/util/git.py

```diff
@@ -76,6 +76,20 @@ def _get_diff_endpoints(self):
         return a, b
 
 
+class LocalMergeBuildDiffer(Differ):
+    def __init__(self, repo):
+        self.repo = repo
+        self._check_environment()
+
+    def _check_environment(self):
+        assert len(self.repo.active_branch.commit.parents) == 2
+
+    def _get_diff_endpoints(self):
+        a = self.repo.active_branch.commit.parents[0]
```

Comment:
one thought... do we know that the parents of a commit are ordered in a consistent manner? if not, may want to add redundancy in the form of seeing whether one of the commits was on master branch and the other was not and ordering them that way.

Reply:
I used this stack overflow to confirm that the ordering is exact for two-parent merges: https://stackoverflow.com/questions/50432112/merge-commit-first-and-second-parents

I'll look to see if I can reference something in the git documentation directly

Reply:
that seems convincing enough to me unless you find something easily in git documentation. please do add a comment/docstring with a brief explanation after.